test/framework/easyblock.py

```diff
@@ -854,6 +855,53 @@ def test_gen_dirs(self):
         sys.stdout = stdoutorig
         eb.close_log()
 
+    def test_make_builddir(self):
+        """Test make_dir method."""
+        self.contents = '\n'.join([
+            'easyblock = "ConfigureMake"',
+            "name = 'pi'",
+            "version = '3.14'",
+            "homepage = 'http://example.com'",
+            "description = 'test easyconfig'",
+            "toolchain = {'name': 'dummy', 'version': 'dummy'}",
```

Comment:
A bit of nitpicking, should version be '' or dummy here?

Reply:
It doesn't matter at all here, since no `dependencies`

Reply:
Yes, but since we're trying to move away from using 'dummy' as version...

Reply:
We intend to get rid of `dummy` as a *toolchain* for EasyBuild v4.0, see https://github.com/easybuilders/easybuild/issues/447 + https://github.com/easybuilders/easybuild-framework/issues/1181 + https://github.com/easybuilders/easybuild-framework/pull/1233.

But it's better to do that all at once, little point in trying to avoid use of `dummy` in tests for EasyBuild 3.x